src/main/kotlin/utils/CoverageFileStatistics.kt

```diff
@@ -0,0 +1,64 @@
+package utils
+import org.xml.sax.Attributes
+import org.xml.sax.InputSource
+import org.xml.sax.helpers.DefaultHandler
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import javax.xml.parsers.SAXParserFactory
+import kotlin.math.min
+
+object CoverageFileStatistics {
+
+    private class Handler() : DefaultHandler() {
+
+        var currentPrefix = ""
+        var coveredMethods = 0
+        var missedMethods = 0
+        var coveredByPrefix = HashMap<String, Int>()
+        var missedByPrefix = HashMap<String, Int>()
+
+        override fun resolveEntity(publicId: String?, systemId: String?): InputSource {
+            return InputSource(ByteArrayInputStream(byteArrayOf()))
```

Comment:
please add a comment why we do this (I assume otherwise something crashes?) and how this differs from the default behavior of DefaultHandler

Reply:
I have no idea. Shameless clone of your code :-)

Reply:
Ok found it. It switches off DTD validation. Documented.